CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.1] 2024-05-31
+
+### Fixed:
+
+- Infinite loop in OTA manifest observation retry logic that occurred
+  when all mailbox slots were full when retries were exhausted.
```

Comment:
```suggestion
  when retries were exhausted.
```
I think mentioning the mailbox is a little too much detail. Filling up the mailbox can be a cause of observations failing and therefore triggering retries, but it's the exhaustion of retries, regardless of cause, that caused the bug.